src/components/molecules/Visualizer/Plugin/api.ts

```diff
@@ -188,6 +197,46 @@ export function exposed({
             return commonReearth.plugins.instances;
           },
         },
+        clientStorage: {
+          get getAsync() {
+            return (key: string) => {
+              const promise = clientStorage.getAsync(plugin?.id ?? "", key);
+              promise.finally(() => {
+                startEventLoop?.();
+              });
+              return promise;
+            };
+          },
+          get setAsync() {
+            return (key: string, value: any) => {
+              const localValue =
+                typeof value === "object" ? JSON.parse(JSON.stringify(value)) : value;
```

Comment:
Converse a Proxy to an object as localForage [can't clone it](https://github.com/localForage/localForage/issues/610).